### ISSUE TITLE
Missing identifier for blockquotes (WCAG 1.3.1c)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1221]
Pages 8 and 9 of the presentation contain quotes that are not marked as such.

The creator of the presentation should be given the opportunity to label quotes as such so that they are marked with the element **blockquote** in the presentation view.

https://slidewiki.org/presentation/81127-1/overview-of-accessibility/81127-1/437649-3/?language=en-/slide-719796-3

**Comment by aoelen** [comment]
In the slide editor, it is possible already to create a `blockquote` element:

![image](https://user-images.githubusercontent.com/1567948/81291872-e91ac400-906a-11ea-8947-e1b35c99158d.png)

I consider this issue resolved, if not: please let me know.
